datascience-notebook/Dockerfile

```diff
@@ -30,32 +24,11 @@ RUN apt-get update --yes && \
 # install Julia packages in /opt/julia instead of ${HOME}
 ENV JULIA_DEPOT_PATH=/opt/julia \
     JULIA_PKGDIR=/opt/julia \
-    JULIA_VERSION="${julia_version}"
 
 WORKDIR /tmp
```

Comment:
It's better to move this to the part where it's needed in the bash script.

Reply:
Done!